LoopKit/InsulinKit/DoseStore.swift

```diff
@@ -856,16 +879,16 @@ extension DoseStore {
         getPumpEventDoseEntriesForSavingToHealthStore(startingAt: start) { (result) in
             switch result {
             case .success(let doses):
-                guard doses.count > 0 else {
-                    completion(nil)
+                guard doses.count > 0, let insulinDeliveryStore = self.insulinDeliveryStore as? InsulinDeliveryStore else {
+                    completion(nil) // ANNA TODO
```

Comment:
What is the work for this TODO?

Reply:
I missed that one, sorry. It's a cue that I put in for myself to go back and make sure that it's the right course of action when I finish making my changes